app/domain/logs.rb

```diff
@@ -44,6 +44,11 @@ module Authentication
       code: "CONJ00030D"
     )
 
+    EncodedJWTResponse = ::Util::TrackableLogMessageClass.new(
+      msg:  "Responding with base64 encoded JWT",
```

Comment:
@InbalZilberman is this message ok? do we want something like `Returning a base64 encoded access token`?

Reply:
https://github.com/cyberark/conjur/pull/1710#discussion_r477150181
BTW, @shulifink said it's fine...

Reply:
Im with @orenbm it is not regarding english it is regarding what we are returning. We call it access token in all other places.

Reply:
@liavyona can you change it to `Responding with a base64 encoded access token` and get @shulifink 's approval please?

Reply:
Changed. WDYS @shulifink ?